frontend/conf/routes

```diff
@@ -2,8 +2,8 @@
 # This file defines all application routes (Higher priority routes first)
 # ~~~~
 
-GET            /robots.txt                            controllers.CachedAssets.at(path="/public", file="robots.txt")
-GET            /humans.txt                            controllers.CachedAssets.at(path="/public", file="humans.txt")
+GET            /robots.txt                            controllers.CachedAssets.at(path="/public", file="robots.txt", aggressiveCaching:Boolean ?= false )
```

Comment:
Without the question mark, Play assumes that value will be provided in the left part of the route. Since we don't provide it, it fails with a 500. 

Related resources:

https://www.playframework.com/documentation/2.0.2/ScalaRouting
http://stackoverflow.com/questions/13633089/pass-parameter-value-in-routes